code/modules/vehicles/pioneer_stone.dm

```diff
@@ -0,0 +1,46 @@
+/obj/vehicle/ridden/pioneer_stone
+	name = "pioneer stone"
+	desc = "Pioneer's used to ride these babies for miles"
+	icon_state = "pioneer_stone"
+	max_integrity = 500
+	armor = list("melee" = 75, "bullet" = 0, "laser" = 25, "energy" = 25, "bomb" = -75, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100) //it is a fucking stone, what do you expect
+	are_legs_exposed = TRUE
+	var/max_damage_force = 25
+	fall_off_if_missing_arms = TRUE
+	resistance_flags = FIRE_PROOF | UNACIDABLE | ACID_PROOF | FREEZE_PROOF
+
+/obj/vehicle/ridden/pioneer_stone/Initialize()
+	. = ..()
+	var/datum/component/riding/D = LoadComponent(/datum/component/riding)
+	D.set_riding_offsets(RIDING_OFFSET_ALL, list(TEXT_NORTH = list(0, 4), TEXT_SOUTH = list(0, 4), TEXT_EAST = list(0, 4), TEXT_WEST = list( 0, 4)))
+	D.vehicle_move_delay = 2
+
+/obj/vehicle/ridden/pioneer_stone/Moved()
+	. = ..()
+	playsound(src, 'sound/effects/clang.ogg', 25, TRUE)
+	if(has_buckled_mobs())
+		for(var/atom/A in range(0, src))
+			if(!(A in buckled_mobs))
+				Bump(A)
```

Comment:
Does this whole section even need to exist?
If so, loop doesn't need to be typed.

Reply:
wouldn't it throw people around without that if(has_buckled_mobs()) whenever someone tried dragging it

Reply:
Bump gets called regardless of this blocks existance. At least it should.
If it doesn't, consider not putting this behavior on Bump at all.

Reply:
actually thats true ill try to build it all without the whole block and see if it is needed, as i copied that code from speedwagon

Reply:
it works! damn nemvar thanks a lot